incubator/hnc/internal/forest/namespace.go

```diff
@@ -27,6 +27,10 @@ type Namespace struct {
 	exists               bool
 	allowCascadingDelete bool
 
+	// labels store the original namespaces's labels, and we add the propogate label to identify itself
+	// in the exception process
+	labels map[string]string
```

Comment:
Do we store all the labels in the forest or just the propagate labels?

Reply:
Changed to store all the labels